.circleci/config.yml

```diff
@@ -688,6 +695,7 @@ jobs:
   create_pullrequest_for_modelupdate:
     docker:
       - image: circleci/android:api-27-alpha
+    working_directory: /tmp/project
```

Comment:
this required?

Reply:
it is required for `git clone` to work properly.